flink-optimizer/src/main/java/org/apache/flink/optimizer/plantranslate/JobGraphGenerator.java

```diff
@@ -106,9 +106,9 @@
  * are created for the plan nodes, on the way back up, the nodes connect their predecessors.
  */
 public class JobGraphGenerator implements Visitor<PlanNode> {
-	
+
 	public static final String MERGE_ITERATION_AUX_TASKS_KEY = "compiler.merge-iteration-aux";
-	
```

Comment:
this would be a lot easier to review without the formatting changes :/

Reply:
Sorry for that. Forgot to disable stripping whitespaces on save. Will revert those changes.